app/forms/concerns/waste_exemptions_engine/can_create_address_from_finder_data.rb

```diff
@@ -11,7 +11,7 @@ def create_address(selected_address_uprn, type)
         data = temp_addresses.detect { |address| address["uprn"] == selected_address_uprn.to_i }
         return unless data
 
-        address = TransientAddress.create_from_address_finder_data(data, type)
+        address = TransientAddress.create_from_address_finder_data(data, TransientAddress.address_types[type])
```

Comment:
Just FYI: https://github.com/DEFRA/waste-exemptions-engine/pull/383/files#diff-04df5204897e7bf26c6002f14ff9e658 :D

Reply:
Well we'll just see who gets there first!

Reply:
That's unfair! It depends on you reviewing my PRs XD